deps.bzl

```diff
@@ -74,8 +74,9 @@ filegroup(
     name = "load_files",
     srcs = [
         "load.bash",
-        "src/assert.bash",
-    ],
+    ] + glob([
```

Comment:
Is there an issue with globs?

Reply:
Yes, explicit declarations are easier to understand. I would avoid where not necessary.

Reply:
I see. The issue is that different versions of the library contain different files. I explicitly mentioned the file that actually gets imported via the `load` command (`load.bash`).

Reply:
`bazel-bats` uses a fixed commit ID for the `bats-core` library, so you should be OK to name the files explicitly regardless of how `bats-core` evolves.

But I don't want to make it overly complex for you to contribute either, so if you think this is easier, feel free to leave as-is and close this and the remark below.

Reply:
> `bazel-bats` uses a fixed commit ID for the `bats-core` library

This is for `bats-support` and `bats-assert`. In any case, `bazel-bats` also exposes parameters for the libraries, allowing for the end repo to set the version (which is good, for flexibility, without needing to change this repo). The issue is that if those bats versions add/remove/rename files and cause the BUILD file strings to no longer be valid, an update would be necessary in this repo. Using globs would allow for future compatibility with more potential bats changes.

> But I don't want to make it overly complex for you to contribute either, so if you think this is easier, feel free to leave as-is and close this and the remark below.

I don't want to lower any quality of the repo in the name of convenience of adding more code.. I just don't really see an alternative to `glob` to still maintain flexibility for changes in bats versions.

Reply:
I also already see `glob` being used `_BATS_CORE_BUILD` a bunch already, so not sure what the issue would be here.

Reply:
LGTM, feel free to leave as is.